instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+import styled from 'styled-components';
+
+const StyledSection = styled.div`
+    border: none;
+`;
+
+class CommentSection extends React.Component {
```

Comment:
Really cool use of local storage to get your comments to persist through reload.